### PR TITLE
New WinGet package manifest for The Silver Searcher

### PR DESCRIPTION
**Pull request by JFLarvoire**
Submitting The Silver Searcher version 2.2.3.yaml

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/microsoft/winget-pkgs/pull/1576)

### CODE REVIEW

File: manifests/JFLarvoire/Ag/2.2.3.yaml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  Silent: -y
+  SilentWithProgress: -y
+Installers:
+  - Arch: x86
+    Url: https://github.com/JFLarvoire/the_silver_searcher/releases/download/2.2.3-Windows/ag_setup.exe
+    Sha256: 69206e14e933ee8a037d64f0f857b69b9f7da266dbcfd39a4d10a485d80a2f73
+  - Arch: x64
+    Url: https://github.com/JFLarvoire/the_silver_searcher/releases/download/2.2.3-Windows/ag_setup.exe
+    Sha256: 69206e14e933ee8a037d64f0f857b69b9f7da266dbcfd39a4d10a485d80a2f73
+ManifestVersion: 0.1.0
```
Comment (Berkeley-Reject) [inline manifests/JFLarvoire/Ag/2.2.3.yaml:36]: I think the `ManifestVersion` field is not supported yet so you could remove this line in order to pass the manifest validation.

Comment (JFLarvoire) [reply manifests/JFLarvoire/Ag/2.2.3.yaml]: OK, done, I've just pushed a change removing it.
Note that the command 'winget validate ...' did report "Manifest validation succeeded."
So there is an inconsistency between that and the bot validation code.

**Comment by JFLarvoire** [comment]
The validation has started 5 hours ago, and is obviously hung.
What can we do to diagnose what's going wrong?

File: manifests/JFLarvoire/Ag/2.2.3.yaml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+Switches:
+  Silent: -y
+  SilentWithProgress: -y
+Installers:
+  - Arch: x86
+    Url: https://github.com/JFLarvoire/the_silver_searcher/releases/download/2.2.3-Windows/ag_setup.exe
+    Sha256: 69206e14e933ee8a037d64f0f857b69b9f7da266dbcfd39a4d10a485d80a2f73
+  - Arch: x64
+    Url: https://github.com/JFLarvoire/the_silver_searcher/releases/download/2.2.3-Windows/ag_setup.exe
+    Sha256: 69206e14e933ee8a037d64f0f857b69b9f7da266dbcfd39a4d10a485d80a2f73
```
[the remark this answers is not in the document]
Comment (JFLarvoire) [reply manifests/JFLarvoire/Ag/2.2.3.yaml]: Done also

**Comment by JFLarvoire** [comment]
On my system, the `winget install -m 2.2.3.yaml` test command triggers a Windows Defender SmartScreen security popup. Clicking "More Info" then "Run Anyway" allows the installation to complete successfully.
Why is it so?
Because I don't get that security popup when I run the ag_setup.exe command directly.
Could it be the reason why your verification process hung for about 10 hours?

**Comment by JFLarvoire** [comment]
No, you probably missed my previous comment:
As the verification had obviously hung, I experimented with signing the program. Which indeed broke the sha256.
I just reverted the ag_setup.exe binary back to its original version, so the sha256 is now correct again. Please have a look now that this is fixed, and tell me what caused the original validation to hang.

**Review by ghost — APPROVED** [review]
Validation has completed

**Pull request merged**